src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -159,6 +161,23 @@ class Filter extends Component<Props> {
     )
   }
 
+  @track((props: Props, _state, [_selected, category, count]) => {
```

Comment:
Do we want to instrument this for the 'ways to buy' and price range sections? I think this is just for the radio button style category changes. (fine to come later)